Remotion/Development/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBase.cs

```diff
@@ -215,9 +215,9 @@ public void AssertAttribute (XmlNode node, string attributeName, string? attribu
     /// and that the attribute's value is equal to or contains <paramref name="attributeValue"/>, 
     /// depending on <paramref name="mode"/>.
     /// </summary>
-    public void AssertAttribute (XmlNode node, string attributeName, WebString attributeValue, AttributeValueCompareMode mode)
+    public void AssertAttribute (XmlNode node, string attributeName, PlainTextString attributeValue, AttributeValueCompareMode mode)
```

Comment:
Given the base-call, I think we should go with `PlainTextString?` and then pass null instead of empty if the value is null. Of course, that wasn't done with WebString because no one noticed. Chage it now or hope it won't pop up as future work?